pac4j-oauth/src/main/java/org/pac4j/oauth/client/BaseOAuth20StateClient.java

```diff
@@ -0,0 +1,77 @@
+package org.pac4j.oauth.client;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.util.CommonHelper;
+import org.pac4j.oauth.client.BaseOAuth20Client;
+import org.pac4j.oauth.client.exception.OAuthCredentialsException;
+import org.pac4j.oauth.credentials.OAuthCredentials;
+import org.pac4j.oauth.profile.OAuth20Profile;
+import org.scribe.oauth.StateOAuth20Service;
+
+/**
+ * Created by jim on 1/4/16.
```

Comment:
Can you add the appropriate Javadoc with the `@author` and `@since` annotations?